/**
 * The package contains its own exceptions for the Service layer.
 */
package by.training.service.exception;